lua/astrocommunity/pack/wgsl/init.lua

```diff
@@ -0,0 +1,23 @@
+local utils = require "astronvim.utils"
+return {
+  {
+    "szebniok/tree-sitter-wgsl",
+    dependencies = "nvim-treesitter/nvim-treesitter",
+    lazy = false,
```

Comment:
```suggestion
    event = "User AstroFile",
```

not sure if that works for this particular plugin but couldn't you lazy load when actually opening a file buffer?

Reply:
I think instead of that we should just reverse the dependencies. Make treesitter dependent on this package. Then the treesitter module will trigger this lazy loading.

Reply:
I have a suggestion here